extensions/rich-text-extension/package.json

```diff
@@ -1,42 +1,52 @@
 {
   "name": "rich-text-extension",
-  "version": "0.3.6",
+  "version": "1.0.0",
   "private": true,
-  "devDependencies": {
-    "@babel/core": "7.3.4",
-    "@contentful/contentful-extension-scripts": "0.15.1",
-    "@contentful/eslint-config-extension": "0.4.2",
-    "@testing-library/react": "11.2.6",
-    "@types/jest": "26.0.23",
-    "@types/react": "^16.8.17",
-    "@types/react-dom": "^16.8.4",
-    "@types/webpack-env": "1.16.0",
-    "contentful-cli": "1.6.11",
-    "cssnano": "4.1.11",
-    "eslint": "^6.0.1",
-    "typescript": "3.9.7"
+  "dependencies": {
+    "@contentful/app-sdk": "^3.36.0",
+    "@contentful/field-editor-single-line": "^0.14.1",
+    "@contentful/forma-36-fcss": "^0.3.2",
+    "@contentful/forma-36-react-components": "^3.90.1",
+    "@contentful/forma-36-tokens": "^0.10.2",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
   },
   "scripts": {
-    "start": "contentful-extension-scripts start",
-    "build": "contentful-extension-scripts build",
-    "deploy": "npm run build && contentful extension update --force",
-    "configure": "contentful space use && contentful space environment use",
-    "login": "contentful login",
-    "logout": "contentful logout",
-    "help": "contentful-extension-scripts help"
+    "start": "cross-env SKIP_PREFLIGHT_CHECK=true BROWSER=none react-scripts start",
+    "build": "SKIP_PREFLIGHT_CHECK=true GENERATE_SOURCEMAP=false react-scripts build",
```

Comment:
same old :P
```suggestion
    "build": "cross-env SKIP_PREFLIGHT_CHECK=true GENERATE_SOURCEMAP=false react-scripts build",
```

Reply:
Will fix it in this PR: https://github.com/contentful/field-editors/pull/759 :)